lib/parser/runner/ruby_rewrite.rb

```diff
@@ -32,7 +32,10 @@ def setup_option_parsing(opts)
     def load_and_discover(file)
       load file
 
-      const_name = file.
+      file_name = File.basename(file)
+      file_name[0] = file_name[0].capitalize
```

Comment:
@whitequark when using it the gsub section below only seems to capitalize the letter after the underscore. 

Input: string_wrapper.rb
Actual Output: stringWrapper

When adding that line it now outputs the following:
Expected Output: StringWrapper